contracts/standard/arbitration/composed-arbitrable/example/ArbitrableTokenList.sol

```diff
@@ -0,0 +1,594 @@
+     *  @param challenger Address of the challenger, if any.
+     *  @param tokenID The tokenID of the item.
+     *  @param status The status of the item.
+     *  @param disputed The item is being disputed.
+     */
+    event ItemStatusChange(
+        address indexed submitter,
+        address indexed challenger,
+        bytes32 indexed tokenID,
+        ItemStatus status,
+        bool disputed
+    );
+
+    /* Storage */
+
+    // Settings
+    uint public challengeReward; // The challengeReward to put to submit/clear/challenge an item in addition of arbitration fees.
+    uint public timeToChallenge; // The time before which an action is executable if not challenged.
+    address public challengeRewardGovernor; // Is allowed to update challengeReward and challengeRewardGovernor
+
+    // Ruling Options
+    uint8 constant REGISTER = 1;
+    uint8 constant CLEAR = 2;
+
+    // Items
+    mapping(bytes32 => Item) public items;
+    mapping(uint => bytes32) public disputeIDToItem;
+    bytes32[] public itemsList;
+
+    // Agreement and Item extension
+    mapping(bytes32 => uint) public itemIDToAgreementCount;
+    mapping(bytes32 => bytes32) public agreementIDtoItemID;
+
+    /* Constructor */
+
+    /**
+     *  @dev Constructs the arbitrable permission list and sets the type.
+     *  @param _arbitrator The chosen arbitrator.
+     *  @param _arbitratorExtraData Extra data for the arbitrator contract.
+     *  @param _metaEvidence The URL of the meta evidence object.
+     *  @param _challengeReward The amount in Weis of deposit required for a submission or a challenge in addition of the arbitration fees.
+     *  @param _challengeRewardGovernor The challengeReward governor
+     *  @param _timeToChallenge The time in seconds, other parties have to challenge.
+     *  @param _feeGovernor The fee governor of this contract.
+     *  @param _stake The stake parameter for sharing fees.
+     */
+    constructor(
+        Arbitrator _arbitrator,
+        bytes _arbitratorExtraData,
+        string _metaEvidence,
+        uint _challengeReward,
+        address _challengeRewardGovernor,
+        uint _timeToChallenge,
+        address _feeGovernor,
+        uint _stake
+    ) public MultiPartyInsurableArbitrableAgreementsBase(_arbitrator, _arbitratorExtraData, _feeGovernor, _stake){
+        emit MetaEvidence(0, _metaEvidence);
+        challengeReward = _challengeReward;
+        timeToChallenge = _timeToChallenge;
+        challengeRewardGovernor = _challengeRewardGovernor;
+    }
+
+    /* Public */
+
+    /**
+     *  @dev Request for an item to be registered.
+     *  @param _tokenID The tokenID of the item to register.
+     *  @param _metaEvidence The meta evidence for the potential dispute.
+     *  @param _arbitrationFeesWaitingTime The maximum time to wait for arbitration fees if the dispute is raised.
+     *  @param _arbitrator The arbitrator to use for the potential dispute.
```

Comment:
Makes sense. should I also make arbitrationFeesWaitingTime into a changeable contract setting? Otherwise the caller could just set it to 0 to make it harder to raise a dispute.

Reply:
Yes